tests/modules/test_combineobjects.py

```diff
@@ -16,6 +16,11 @@ def image():
 
     return cellprofiler_core.image.Image(data)
 
+@pytest.fixture
+def image_volume():
```

Comment:
Try and use structuring elements (or the random data function we wrote for skimage).

Reply:
You could also put an image in the `tests/data` directory.

Reply:
If I understand the way that these tests are structured correctly, rather than random data, we define specific numpy arrays that we can predict how they should be combined in order to then compare the output from combineobjects (@DavidStirling can confirm, since I think he set up the structure in 2D that I'm following in 3D). I don't think we could use random shapes w/ this approach, but I may be missing something.

Reply:
In 2D we generate some very specific object sets to test that each of the different merge methods works as expected (covering complete overlap, partial overlap, multiple overlap and so on). To recreate this in 3D using the ball strel could be a good approach, just need to be careful about placement. I agree that random shapes probably isn't what we want.

Reply:
We've decided to use the specific object sets. In the future, the ball strel approach would be an option for improvement